src/components/App.scss

```diff
@@ -1 +1,64 @@
+// Define constants
```

Comment:
We can remove the comments as the names used for the constants should be explanatory enough